docs/getting-started.md

```diff
@@ -18,7 +18,7 @@ Creating a new Scalar clone using the GVFS Protocol
 ---------------------------------------------------
 
 The `clone` verb creates a local enlistment of a remote repository using the
-[GVFS protocol](https://github.com/microsoft//VFSForGit/blob/main/Protocol.md).
+[GVFS protocol](https://github.com/microsoft//VFSForGit/blob/HEAD/Protocol.md).

```

Comment:
I'm sorry to bother you again, but could you fix a double slash here, too? `//` seems a bit untidy. Here, and in the `index.md` too.